### DISCUSSION TITLE
How to merge dependabot PR in proper way

### DISCUSSION THREAD

**Discussion by danny007in** [ColorlibHQ/AdminLTE#3332] [General]
I have merged dependabot PR directly by merge and commit like has usually, but is that needed to squash or rebase after merge or how @XhmikosR  @REJack 
Because Messages is showing in commit history

![Screenshot 2021-01-18 173139](https://user-images.githubusercontent.com/50356015/104913086-1fa72d80-59b3-11eb-84b4-982874632ca4.png)

how to remove those messages

**Comment by XhmikosR** [comment]
https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges

But keep in mind that GitHub **will remember** your last option, and when there are cases the patches shouldn't be squashed etc.

There are repo settings to enforce this stuff, but I don't control the repo settings myself, and it might be OK to do a merge commit in some cases.
